util/io/game_state.py

```diff
@@ -22,10 +26,24 @@ def __init__(self, packet: GameTickPacket, previous_game_state, index: int):
         self.team = self.car.team
         self.ball = BallData(packet.game_ball.physics)
 
+    def toFrameworkGameState(self):
+        ball_state = framework_utils.BallState(
+            framework_utils.Physics(location=self.ball.position.to_game_state_vector3(),
+                                    velocity=self.ball.velocity.to_game_state_vector3()))
+        return framework_utils.GameState(ball=ball_state)
+

```

Comment:
here, we take a game state, and we transfer it into the framework's game state type to use for state setting. It's mainly gonna be used to translate the received game state from the opponent, and then state set directly with the resulting state.